### ISSUE TITLE
Logo needs more html structure

### ISSUE THREAD

**Issue by WimTeuling** [MadiMagpie/anna-madison-metrolics#11]
https://github.com/MadiMagpie/anna-madison-metrolics/blob/f8692ed9f6218f0b8ac494f2b45ed541bd7c53e6/index.html#L28

Use text if possible (good for SEO and accessibility). Link is missing.

Compare:

```
<div class="logo logo--metrolics">
    <a href="#" class="logo__link">
        <h2 class="logo__type">metrolics</h2>
        <img src="img/theme/metrolics_logo.svg" alt="Metrolics Logo" class="logo__img">
    </a>
</div>
```
